ql/src/java/org/apache/hadoop/hive/ql/parse/repl/CopyUtils.java

```diff
@@ -33,6 +33,7 @@
 import org.slf4j.LoggerFactory;
 
 import javax.security.auth.login.LoginException;
+import java.io.FileNotFoundException;
```

Comment:
Any manual test done for this scenario where source is not accessible during repl load?

Reply:
yes